packages/dev/core/src/Navigation/Plugins/recastJSPlugin.ts

```diff
@@ -501,6 +523,22 @@ export class RecastJSPlugin implements INavigationEnginePlugin {
     public isSupported(): boolean {
         return this.bjsRECAST !== undefined;
     }
+
+    /**
+     * Returns the seed used for randomized functions like `getRandomPointAround`
+     * @returns seed number
+     */
+    public getRandomSeed(): number {
+        return this.bjsRECAST._getRandomSeed();
+    }
+
+    /**
+     * Set the seed used for randomized functions like `getRandomPointAround`
+     * @param seed number used as seed for random functions
+     */
+    public setRandomSeed(seed: number): void {
+        this.bjsRECAST._setRandomSeed(seed);
+    }

```

Comment:
this could be a single function. Make number optional.

Also - I don't see the definition of this function in the declaration file. Was it always available as a hidden public function?

Reply:
no, it's a C function that is exposed with cmake (hence the _ prefix)